src/aggregate/aggregate_request.c

```diff
@@ -1319,6 +1319,8 @@ static void SafeRedisKeyspaceAccessPipeline(AREQ *req) {
   // root<-buffer-locker<-sorter<-loader<-unlocker
   ResultProcessor *upstream_is_buffer_locker = NULL;
   ResultProcessor *upstream_is_unlcoker = NULL;
+  char wrapsLastAccumulator = 2; // 0 - no, 1 - yes, 2 - unset. 0 and 1 are "terminal states".
```

Comment:
Have those values as an enum, they will still fit into char if you will not exceed 255 values
Also, comments about the purpose